cmd/goclient/goclient.go

```diff
@@ -277,11 +278,11 @@ func main() {
 
 			var amountbytes [8]byte
 			binary.LittleEndian.PutUint64(amountbytes[:], amount)
-			txn := aptos.RawTransaction{
+			txn := types.RawTransaction{
```

Comment:
I think this could make user code less clear. They may already have their own 'types' package. Having Aptos types packaged as 'aptos' is probably better for a developer using the SDK.